test/vanilla-wildfly/test-app/pom.xml

```diff
@@ -48,6 +48,9 @@
                             </plugin-options>
                             <cloud/>
                             <server/>
+                            <extra-server-content-dirs>
```

Comment:
is the `<extra-server-content-dirs>` element bound to the `<server>` mode? or does it work with Bootable Jar too?

Reply:
It is a common support for JAR and server modes. This support is not new, it is there since version 2.0 of the plugin.